test/hotspot/jtreg/compiler/codegen/Test8005033.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012 SAP SE. All rights reserved.
+ * Copyright (c) 2012, 2022 SAP SE. All rights reserved.
```

Comment:
We'd better not change the copyright line other than Oracle's.
Maybe, you can start a newline of yours after it.

Reply:
hi, JIefu, 
Is there a rule for this case? 
I search and found many SAP SE copyright line has [beg, end] years.

Reply:
hi, @DamonFool , 

Now I think I understand your concern. OCA says that contributors share their rights with Oracle.  I am not Oracle. I don't have permission to change other people's copyright headers.  I revert that and use ours. is that okay now? 

thanks,
--lx